libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -52,6 +52,11 @@ export const ReviewCreatePanel = () => {
       id: 'LVeWG7',
       description: 'Accessibility label for state kind',
     }),
+    PLACEHOLDER: intl.formatMessage({
+      defaultMessage: '----',
```

Comment:
Shoudlnt this be 'Enter workflow name'

Reply:
I was thinking we discussed in the meeting to show ---- to be consistent with other parts of azure portal